Tasks/NpmAuthenticate/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "ad884ca2-732e-4b85-b2d3-ed71bcbd2788",
+    "name": "npmAuthenticate",
+    "friendlyName": "npm Authenticate",
+    "description": "Authenticate a .npmrc file for tasks which use npm.",
```

Comment:
Provides npm credentials to an .npmrc file in your repository for the scope of the build. This enables npm task runners like Gulp and Grunt to authenticate with private registries.